modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/IrohaContainer.kt

```diff
@@ -72,8 +74,16 @@ open class IrohaContainer : GenericContainer<IrohaContainer> {
                 config.genesis?.writeToFile(genesisFileLocation)
                 config.genesisPath?.also { path -> Files.copy(Path(path).toAbsolutePath(), genesisFileLocation) }
 
-                getResource(DEFAULT_EXECUTOR_FILE_NAME).readBytes().let { content ->
-                    executorFileLocation.toFile().writeBytes(content)
+                if (config.useLocalTestExecutor) {
```

Comment:
why can't we use the same way genesisPath is handled, copying the specified file to the target default location of the executor?